1주차/14659/kotlin_14659_페퍼.kt

```diff
@@ -0,0 +1,22 @@
+import kotlin.math.max
+
+fun main(args: Array<String>) {
+    val n = readln().toInt()
+    val archers = readln().split(" ").map { it.toInt() }
+    var maxCount = 0
+
+    var idx = 1
+    var archer = archers[0]
+    var cnt = 0
+    while (idx < n) {
```

Comment:
```suggestion
    for (archer in archers) {
```
for 문을 사용하면 Index를 사용하지 않아도 돼서 더 편할 것 같아요!

Reply:
전 공격한 봉우리의 개수를 기준으로 `idx`를 변경해서 필요하다고 생각했는데, 봉우리의 높이를 기준으로 생각하면 `idx`가 불필요했네요ㅎㅎ 파랑의 풀이법도 하나 배워갑니당ㅎㅎ 🥳🥳